### PR TITLE
C1X Adapter GDPR Support

### PR DESCRIPTION
**Pull request by CathyHuangtw**
## Type of change
- [x] Feature

## Description of change
Adding GDPR support to C1X adapter

contact email of the adapter’s maintainer
cathy@c1exchange.com

### CODE REVIEW

**Comment by CathyHuangtw** [comment]
I've also sent a PR to update our doc on Prebid website
[PR to prebid.github.io](https://github.com/prebid/prebid.github.io/pull/871)

File: modules/c1xBidAdapter.js [hunk 1]
```diff
@@ -48,15 +49,30 @@ export const c1xAdapter = {
       response: 'json',
       compress: 'gzip'
     }
-    Object.assign(payload, tagObj);
 
-    let payloadString = stringifyPayload(payload);
+    if (pixelId) {
```
Comment (mike-chowla) [inline modules/c1xBidAdapter.js]: Why is there sync added here, when the sync is also added on line 59?  Adding two syncs seems wasteful.

**Comment by CathyHuangtw** [comment]
@mike-chowla  Thanks for pointing that out. I only intended to do one sync but forgot deleting the old codes. I have updated the code and ensure there is only one user sync.

**Comment by mike-chowla** [comment]
@CathyHuangtw Please create a doc PR to update that your adapter now supports GDPR consent passing

**Comment by CathyHuangtw** [comment]
@mike-chowla  Thanks for your review.
I've already created a doc PR [here](https://github.com/prebid/prebid.github.io/pull/871).

**Pull request merged**